sass/main.scss

```diff
@@ -680,121 +680,304 @@ nav {
+
+        // Sequentially assign colors to all elements
+        .event:nth-child(3n) {
+            border-top: 10px solid $red;
+        }
+        .event:nth-child(3n + 1) {
+            border-top: 10px solid $orange;
+        }
+        .event:nth-child(3n + 2) {
+            border-top: 10px solid $light-gray;
+        }
+
+        .event {
+            background: white;
+            border-radius: 15px;
+            margin: 10px 0px;
+            padding: 10px;
+            position: relative;
+
+            &.past {
+                background-color: #bfbfbf;
+            }
+
+            .time {
+                color: $light-blue;
+            }
+
+            .title {
+                color: black;
+                font-size: 1.3em;
+            }
+
+            .marker {
+                $marker-width: 20px;
 
-                    border-bottom: 2px solid $red;
-                    border-top: 2px solid $red;
+                border-bottom: 2px solid $red;
+                border-top: 2px solid $red;
+                position: absolute;
+                left: -25px;
+                right: 0;
+                top: 0;
+
+                &::before {
+                    content: "";
+                    background-color: $red;
+                    border-radius: 50%;
                     position: absolute;
-                    left: -25px;
-                    right: 0;
-                    top: 0;
-
-                    &::before {
-                        content: "";
-                        background-color: $red;
-                        border-radius: 50%;
-                        position: absolute;
-                        bottom: -($marker-width / 2);
-                        left: -($marker-width / 2);
-                        height: $marker-width;
-                        width: $marker-width;
-                    }
+                    bottom: -($marker-width / 2);
+                    left: -($marker-width / 2);
+                    height: $marker-width;
+                    width: $marker-width;
                 }
             }
         }
+    }
+
+    // Note that pre-event style overrides default schedule content styling
+    #pre-event-content {
+        display: flex;
+        flex-direction: column;
+        flex-wrap: none;
+
+        .day-row {
+            display: flex;
+            justify-content: space-between;
+        }
+
+        // Coordinate color with day on left and event border on top
+        .pre-16 {
+            .event {
+                border-top-color: $red;
+            }
+
+            h2, h6 {
+                color: $red;
+            }
+        }
+
+        .pre-17 {
+            .event {
+                border-top-color: $red;
+            }
+
+            h2, h6 {
+                color: $orange;
+            }
+        }
+
+        .pre-18 {
+            .event {
+                border-top-color: $red;
+            }
 
+            h2, h6 {
+                color: $orange;
```

Comment:
If I remember correctly, this is supposed to be a different orange.

Also all of the .event border tops are red? Do you want to change them all at once or actually coordinate them in this pr?

Reply:
Good catch! Yeah May also pointed this out on call haha